### PR TITLE
Praetor suit nerf

### PR DESCRIPTION
**Pull request by BobJoga**
## About The Pull Request

Title.

**Old armor stats:**
armor = list("melee" = 75, "bullet" = 55, "laser" = 55, "energy" = 45, "bomb" = 100, "bio" = 100, "rad" = 100, "fire" = 100, "acid" = 100)

**New armor stats:**
armor = list("melee" = 75, "bullet" = 30, "laser" = 30, "energy" = 25, "bomb" = 50, "bio" = 100, "rad" = 50, "fire" = 100, "acid" = 100)

**Captain's hardsuit armor stats, for comparison:**
armor = list("melee" = 40, "bullet" = 50, "laser" = 50, "energy" = 25, "bomb" = 50, "bio" = 100, "rad" = 50, "fire" = 100, "acid" = 100)

## Why It's Good For The Game

Miner powercreep yada yada i'm stuff

## Changelog
:cl:
balance: Nerfed the praetor suit to be comparable to the captain's hardsuit.
/:cl:

### CODE REVIEW

**Comment by YakumoChen** [comment]
put some slowdown on it too to make it comparable to the other lavaland suits. Jesus christ this thing is OP

**Comment by CoreFlare** [comment]
it's a miner drop. if it has slowdown it's useless for all but crusher players.

**Comment by Echopsychotic** [comment]
I agree with @yakumochen give it slowdown as well, so it can be either as a lavaland suit or as a space suit if it acts like a space suit.

**Comment by BobJoga** [comment]
> put some slowdown on it too to make it comparable to the other lavaland suits. Jesus christ this thing is OP
> 
> Baffling why someone would merge a suit that is literally better than deathsquad armor.

> I agree with @YakumoChen give it slowdown as well, so it can be either as a lavaland suit or as a space suit if it acts like a space suit.

No. Slowdown makes it useless for miner. You get this.

**Comment by Echopsychotic** [comment]
Is the suit space proof?

**Comment by BobJoga** [comment]
> Is the suit space proof?

Yes.

File: modular_skyrat/code/modules/mining/lavaland/necropolis_chests.dm [hunk 1]
```diff
@@ -351,7 +351,7 @@
 /obj/item/clothing/suit/space/hardsuit/deathsquad/praetor
 	name = "Praetor Suit"
 	desc = "And those that tasted the bite of his sword named him... The Doom Slayer."
-	armor = list("melee" = 75, "bullet" = 55, "laser" = 55, "energy" = 45, "bomb" = 100, "bio" = 100, "rad" = 100, "fire" = 100, "acid" = 100)
+	armor = list("melee" = 75, "bullet" = 40, "laser" = 40, "energy" = 35, "bomb" = 50, "bio" = 100, "rad" = 50, "fire" = 100, "acid" = 100)
```
Comment (ForrestWick) [inline modular_skyrat/code/modules/mining/lavaland/necropolis_chests.dm:354]: armor = list("melee" = 55, "bullet" = 30, "laser" = 20, "energy" = 15, "bomb" = 60, "bio" = 100, "rad" = 50, "fire" = 100, "acid" = 100)

Comment (ForrestWick) [reply modular_skyrat/code/modules/mining/lavaland/necropolis_chests.dm:354]: This is my suggestion, as this thing is way too fucking powerful even with the nerf.

Comment (BobJoga) [reply modular_skyrat/code/modules/mining/lavaland/necropolis_chests.dm:354]: No lmao that is literally worse than other megafauna armor.

**Pull request closed without merging**